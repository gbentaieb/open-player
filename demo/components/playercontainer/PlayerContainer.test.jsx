import React from 'react';
import PlayerContainer from './PlayerContainer';

describe('Test demo player container', () => {
  test('PlayerContainer > Snapshot', () => {
    const wrapper = shallow(<PlayerContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
