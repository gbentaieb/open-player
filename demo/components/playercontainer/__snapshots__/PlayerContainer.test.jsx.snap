// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test demo player container PlayerContainer > Snapshot 1`] = `
<div>
  <div />
</div>
`;
